src/Illuminate/Foundation/Console/ListenerMakeCommand.php

```diff
@@ -54,7 +54,8 @@ protected function buildClass($name)
         $event = $this->option('event');
 
         if (! Str::startsWith($event, $this->laravel->getNamespace()) &&
-            ! Str::startsWith($event, 'Illuminate')) {
+            ! Str::startsWith($event, 'Illuminate') &&
+            ! Str::startsWith($event, '\\')) {
```

Comment:
It was called 2 times, so I figured this is a standard practice in Laravel...
Could we call it with anonymous array, to signify that it will not be used anywhere else?
```
if (! Str::startsWith($event, [
    $this->laravel->getNamespace(),
    'Illuminate',
    '\\',
]) {
// ...
```